### PR TITLE
Doc for subswap

### PR DESCRIPTION
**Pull request by GalaxySciTech**

### CODE REVIEW

**Review by wanwiset25 — APPROVED** [review]

File: applications/subswap/contract/README.md [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
+### Method 1: Using Hardhat Scripts
 
-1. Install Hardhat and dependencies:
+1. **Install Hardhat and Dependencies**
 
-```
-yarn
-```
+   ```bash
+   yarn
```
Comment (wjrjerome) [inline applications/subswap/contract/README.md:24]: missing the installation cmd

File: applications/subswap/contract/README.md [hunk 2]
```diff
@@ -1,32 +1,42 @@
...
+
+## Deploying the Subnet Token
+
+```bash
+npx hardhat run scripts/simpletokendeploy.js --network xdcsubnet
 ```
-npx hardhat run scripts/subnettreasurydeploy.js --network xdcsubnet
-```
+
+This guide provides the steps to check and configure necessary values in the `deploy.config.json` file, and then deploy Subswap and the Subnet Token using Hardhat scripts.
```
Comment (wjrjerome) [inline applications/subswap/contract/README.md]: shouldn't this be at the top of the file

File: applications/subswap/contract/contracts/SimpleToken.sol
Comment (wjrjerome) [inline applications/subswap/contract/contracts/SimpleToken.sol:1]: Question. Is this file relevant to documentations?

File: applications/subswap/contract/scripts/simpletokendeploy.js [hunk 3]
```diff
@@ -5,15 +5,24 @@
...
 
 async function main() {
-  const factory = await hre.ethers.getContractFactory("TreasuryToken");
+  const factory = await hre.ethers.getContractFactory("SimpleToken");
 
-  const treasuryToken = await factory.deploy("test", "test");
+  const token = deploy.subnettoken;
 
-  await treasuryToken.deployed();
+  console.log(token);
```
Comment (wjrjerome) [inline applications/subswap/contract/scripts/simpletokendeploy.js]: to be removed?

File: applications/subswap/contract/scripts/simpletokendeploy.js [hunk 4]
```diff
@@ -5,15 +5,24 @@
 // will compile your contracts, add the Hardhat Runtime Environment's members to the
 // global scope, and execute the script.
 const hre = require("hardhat");
+const deploy = require("../deploy.config.json");
 
 async function main() {
```
Comment (wjrjerome) [inline applications/subswap/contract/scripts/simpletokendeploy.js:10]: just an improvement. I think we shall have try catch with some nice error message in case of failure

File: applications/subswap/README.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#Subswap
```
Comment (wgr523) [inline applications/subswap/README.md]: space after `#`

File: applications/subswap/README.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+
+Contract Deploy doc in [contract](contract/)
+
+Frontend doc in [contract](frontend/)
```
Comment (wgr523) [inline applications/subswap/README.md]: 1. Frontend doc in [frontend], not [contract]
2. can you directly use url of https://github.com/XinFinOrg/subswap-frontend

File: applications/subswap/contract/contracts/SimpleToken.sol
Comment (GalaxySciTech) [reply applications/subswap/contract/contracts/SimpleToken.sol:1]: This file relative to doc for how to deploy new token

File: applications/subswap/contract/README.md (hunk 1 above)
Comment (GalaxySciTech) [reply applications/subswap/contract/README.md:24]: yarn is install cmd

File: applications/subswap/contract/README.md (hunk 2 above)
Comment (GalaxySciTech) [reply applications/subswap/contract/README.md]: done
